src/main/java/com/denizenscript/ddiscordbot/commands/DiscordInteractionCommand.java

```diff
@@ -0,0 +1,564 @@
+                result = CommandPrivilege.enable(DiscordRoleTag.valueOf(item, scriptEntry.getContext()).role);
+            }
+            if (result != null) privileges.add(result);
+        }
+    }
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public void execute(ScriptEntry scriptEntry) {
+        ElementTag id = scriptEntry.getElement("id");
+        ElementTag instruction = scriptEntry.getElement("instruction");
+        ElementTag commandInstruction = scriptEntry.getElement("command_instruction");
+        DiscordGroupTag group = scriptEntry.getObjectTag("group");
+        ElementTag name = scriptEntry.getElement("name");
+        ElementTag description = scriptEntry.getElement("description");
+        List<MapTag> options = (List<MapTag>) scriptEntry.getObjectTag("options");
+        ElementTag enabled = scriptEntry.getElement("enabled");
+        ListTag enableFor = scriptEntry.getObjectTag("enable_for");
+        ListTag disableFor = scriptEntry.getObjectTag("disable_for");
+        DiscordInteractionTag interaction = scriptEntry.getObjectTag("interaction");
+        ElementTag ephermal = scriptEntry.getElement("ephermal");
+        ElementTag attachFileName = scriptEntry.getElement("attach_file_name");
+        ElementTag attachFileText = scriptEntry.getElement("attach_file_text");
+        List<ListTag> rows = (List<ListTag>) scriptEntry.getObjectTag("rows");
+        ElementTag message = scriptEntry.getElement("message");
+        if (scriptEntry.dbCallShouldDebug()) {
+            Debug.report(scriptEntry, getName(), id.debug()
+                    + instruction.debug()
+                    + (commandInstruction != null ? commandInstruction.debug() : "")
+                    + (group != null ? group.debug() : "")
+                    + (name != null ? name.debug() : "")
+                    + (description != null ? description.debug() : "")
+                    + (options != null ? ArgumentHelper.debugList("Options", options) : "")
+                    + (enabled != null ? enabled.debug() : "")
+                    + (enableFor != null ? enableFor.debug() : "")
+                    + (disableFor != null ? disableFor.debug() : "")
+                    + (interaction != null ? interaction.debug() : "")
+                    + (ephermal != null ? ephermal.debug() : "")
+                    + (attachFileName != null ? attachFileName.debug() : "")
+                    + (attachFileText != null ? attachFileText.debug() : "")
+                    + (rows != null ? ArgumentHelper.debugList("Rows", rows) : "")
+                    + (message != null ? message.debug() : ""));
+        }
+
+        DiscordInteractionInstruction instructionEnum = DiscordInteractionInstruction.valueOf(instruction.asString().toUpperCase());
+        if (!DenizenDiscordBot.instance.connections.containsKey(id.asString())) {
+            Debug.echoError("Failed to process DiscordMessage command: unknown bot ID!");
```

Comment:
Wrong command name in this message.

Also more stray newlines a few lines up and down.